onelogin_aws_cli/credentials.py

```diff
@@ -18,28 +19,18 @@ class MFACredentials(object):
     """
 
     def __init__(self):
-        self._device_index = None
-        self._devices = []
-        self._otp = None
```

Comment:
We're still using `self._otp` and `self._devices`, so they probably shouldn't be deleted.

Reply:
These are set by the `reset` call just below.

Reply:
For me, it's a code style/maintenance/readability thing (and happy to take other styles). Being able to look at the init function and see what the instance variables are on an object can be helpful. I did deliberate over how to handle the reset method a couple of times.

That said, this is not a hill I'm willing to die on; just a personal preference.

Reply:
Fair point ... I will admit that I also winced a little when I removed those lines =/